File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/TransportersUpdateDriversRequest.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class TransportersUpdateDriversRequest
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("DriversLicenseNumber")]
        public string? DriversLicenseNumber { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("EmployeeId")]
        public string? EmployeeId { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Id")]
        public string? Id { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("Name")]
        public string? Name { get; set; }
    }
}

File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/TransportersUpdateDriversRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class TransportersUpdateDriversRequest {
    @JsonProperty("DriversLicenseNumber")
    public String driversLicenseNumber;
    @JsonProperty("EmployeeId")
    public String employeeId;
    @JsonProperty("Id")
    public String id;
    @JsonProperty("Name")
    public String name;
}
